### PR TITLE
Building with Active Record Project

### PR DESCRIPTION
**Pull request by gasb150**
In this project we create a rails app using active record.

We didn't build a front end for it because we don’t need to.

We use the Rails console to play around with models without making HTTP requests and involving controllers or views.

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__So far, you've done really well. The project is almost complete, although some changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] You committed the `node_modules` directory to your repository. The `node_modules` directory should not be pushed to your repo. 
- To ensure this, you have to delete the directory in your working branch, commit and push your changes, then add this directory to your `.gitignore` file. 
- You can find more details about the `.gitignore` file [here](https://www.pluralsight.com/guides/how-to-use-gitignore-file). 
- You can also check this STO thread about [removing a directory from git](https://stackoverflow.com/questions/6313126/how-to-remove-a-directory-from-git-repository).

- [x] Ensure that validations are present for the `Post` model (title & description)
- [x] You mentioned in your Readme that the project was built with Ruby v2.7.2, but your Gemfile specified 3.0.0. This can lead to confusion, please use a specific Ruby version and stick to it across your application.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by NtwaliHeritier — APPROVED** [review]
### STATUS: APPROVED✔✔
Great work correcting changes pointed out by the previous TSE.
You can merge the branches

**Pull request merged**
